### PR TITLE
fixes_for_camcan

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: 01-import_and_maxfilter.py [hunk 1]
```diff
@@ -236,6 +236,10 @@ def run_maxwell_filter(subject, session=None):
             raw_fname_out = op.join(config.deriv_root, subject_path,
                                     bids_basename + '_nosss_raw.fif')
 
+        subject_dirname = os.path.dirname(raw_fname_out)
+        if not os.path.exists(subject_dirname):
+            os.mkdir(subject_dirname)
```
Comment (hoechenberger) [inline 01-import_and_maxfilter.py:241]:
```suggestion
        os.makedirs(subject_dirname, exist_ok=True)
```
Saves a line ;)

**Comment by hoechenberger** [comment]
Not sure why CI is failing, but I also have a few changes / fixes for the study template lined up here, maybe one of them fixes CI already. Will take a look tomorrow

**Comment by agramfort** [comment]
I'll use this branch over night. I'll let you take over afterwards

**Comment by agramfort** [comment]
it ran fine. Merging this.

**Pull request merged**